CHANGELOG.md

```diff
@@ -29,6 +29,23 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
 
 ### Linter
 
+### Bug fixes
+
+- Biome no longer crashes when it encounters a string that contain a multibyte character ([#4181](https://github.com/biomejs/biome/issues/4181)).
+
+  This fixes a regression introduced in Biome 1.9.3
+
+  This affects the following linter rules:
+
+  - nursery/useSortedClasses
+  - nursery/useTrimStartEnd
+  - style/useTemplate
+  - suspicious/noMisleadingCharacterClass
+
+  If you are using Biome 1.9.3, yo ucan disable these rules as a temporary workaround.
```

Comment:
```suggestion
  If you are using Biome v1.9.3 or earlier, you can disable these rules as a temporary workaround.
```


I'm not sure I understand the purpose of "the workaround". Once we ship the next fix, users can upgrade to fix the issue, so the workaround isn't needed. Did I miss something?

Reply:
You are right. This could be confusing. I will remove it.